metadatastore/core_v0.py

```diff
@@ -369,7 +369,8 @@ def get_events_generator(descriptor, event_col, descriptor_col,
     col = event_col
     oid = _UID_TO_OID_MAP[descriptor_uid]
     ev_cur = col.find({'descriptor_id': oid},
-                      sort=[('time', pymongo.ASCENDING)])
```

Comment:
These documents do not have a `descriptor` field.  Mongo will use the compound index here automatically ( https://docs.mongodb.org/manual/tutorial/sort-results-with-indexes/#sort-and-index-prefix ).  Once you have the index added, sorting on just time here is ok.